src/notifications.js

```diff
@@ -22,7 +23,7 @@ export default class NotificationsManager {
     }
   }
 
-  static getRelatedCodeErrHandler(filename, lineNo) {
+  static getRelatedCodeErrHandler(filename, lineNo, fileName) {
```

Comment:
I don't think we need the third argument here--we pass in the same as the first (maybe rename that first arg to fileName for consistency with the vscode API if that was the source of confusion).